examples/ng_networking/Makefile

```diff
@@ -0,0 +1,29 @@
+# name of your application
+APPLICATION = ng_networking
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Specify the modules that are used
+USEMODULE += ng_netif
+USEMODULE += ng_sixlowpan_frag
+USEMODULE += ng_ipv6
+USEMODULE += ng_udp
+USEMODULE += ng_pktdump
+USEMODULE += uart0
```

Comment:
on some boards yes. So I rather keep it in until we remove it completely from RIOT...